bundle/docsgen/output/reference.md

```diff
@@ -346,6 +354,10 @@ Defines bundle deployment presets. See [\_](/dev-tools/bundles/deployment-modes.
   - Type
   - Description
   
+- - `artifacts_dynamic_version`
```

Comment:
[optional] it would be easier to review this if catch-up regenerate with unrelated accumulated changes was sent as a separate PR first.